lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -166,7 +166,9 @@
   kernel_memory:
     description:
       - "Kernel memory limit (format: <number>[<unit>]). Number is a positive integer.
```

Comment:
`<number>[<unit>]` should be formatted with `C(...)` as well (everywhere). Currently, backticks are only used for `shm_size` (which should be converted to `C(...)` too).